CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## UNRELEASED
 
+## 1.8.0-beta2 (May 21, 2020)
+
 BUGFIXES:
```

Comment:
These two entries are unreleased, the changelog entries for beta2 are here: https://github.com/hashicorp/consul/blob/release/1.8.x/CHANGELOG.md